pkg/apis/aerogear/v1alpha1/mobileclient_types.go

```diff
@@ -0,0 +1,51 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	MobileClientType = metav1.TypeMeta{
+		APIVersion: "mobile.k8s.io/v1alpha1",
```

Comment:
The APIVersion has been changed in the lasted version of MDC. This should be `mdc.aerogear.org/v1alpha1` now.